quality_control/views/qc_trigger_view.xml

```diff
@@ -28,12 +30,24 @@
         <field name="name">qc.trigger.tree</field>
         <field name="model">qc.trigger</field>
         <field name="arch" type="xml">
-            <tree string="Quality control triggers" editable="bottom">
+            <tree string="Quality control triggers">
```

Comment:
This shouldn't be removed.

Reply:
When editable is set, the view change button is not visible and puts the tree view in edit mode when clicking instead of opening form view.
Am I missing something here?

Reply:
Yes, and that's the desired behavior for me.

Reply:
If you want to make it different on your new menu, create a new view for that menu.

Reply:
@pedrobaeza I didn't mean to be rude.
I just thought there was not use being able to edit just the trigger name in list view.
Anyway, I made the requested change.

Reply:
No worry, I don't think you were rude. Just debating :)

My point is that on product form, the trigger should be an editable tree, or you will lose usability. That's why I want to preserve this.

Reply:
I totally agree with you but qc_triggers defines its tree view in product_template_qc_form_view inline.
In any case I see your point, if the inline definition of view is removed in the future, the tree in product view should be editable. Better play it safe.

Reply:
What do you think about use?
`primary mode`
`sequence > 16`
`<tree position="attributes">`
 ` <attribute name='editable'></attribute>`
`</tree>`